static/bidder-info/zeta_global_ssp.yaml

```diff
@@ -1,5 +1,8 @@
-endpoint: https://ssp.disqus.com/bid/prebid-server?sid=GET_SID_FROM_ZETA
+enabled: false
```

Comment:
Can you please help me understand why you have set `enabled: false`? As per the Prebid Server Documentation, if you want to disable the bidder, you need to set `disabled: true`

Reply:
Seems like copy-pasted from somewhere. Fixed. Thank you!